tutorials/hyperspectral/Advanced_Spectral_Signatures.Rmd

```diff
@@ -0,0 +1,291 @@
+---
+title: "Advanced Spectral Signatures in R"
+code1: HYPERSPECTRAL/Advanced_Spectral_Signatures.R
+contributors: null
+output:
+  html_document:
+    df_print: paged
+dateCreated: 2020-02-18 20:18:34
+description: Plot the spectral signatures of multiple different land cover types using
+  an interactive click-to-extract interface.
+estimatedTime: 0.5 Hours
+languagesTool: R
+dataProudct: null
+packagesLibraries: rhdf5, raster, plyr, reshape2, ggplot2
+syncID: db9715ca243944fabbe81031f2ed5cec
+authors: Donal O'Leary
+topics: hyperspectral, HDF5, remote-sensing
+tutorialSeries: null
+urlTitle: advanced-spectral-signatures-in-r
+---
+
+
+In this tutorial, we will learn how to plot spectral signatures of several
+different land cover types using an interactive clicking feature of the 
+`raster` package.
+
+<div id="ds-objectives" markdown="1">
+
+## Learning Objectives
+
+After completing this activity, you will be able to:
+
+* Extract and plot spectra from an HDF5 file.
+* Work with groups and datasets within an HDF5 file.
+* Use the `raster::click()` function to interact with an RGB raster image
+
+## Things You’ll Need To Complete This Tutorial
+To complete this tutorial you will need the most current version of R and, 
+preferably, RStudio loaded on your computer.
+
+### R Libraries to Install:
+
+* **rhdf5**: `install.packages("BiocManager")`, `BiocManager::install("rhdf5")`
+* **raster**: `install.packages('raster')`
+* **rgdal**: `install.packages('rgdal')`
+* **plyr**: `install.packages('plyr')`
+* **reshape2**: `install.packages('rehape2')`
+* **ggplot2**: `install.packages('ggplot2')`
+
+<a href="https://neonscience.org/packages-in-r" target="_blank"> More on Packages in
+ R - Adapted from Software Carpentry.</a>
+
+### Data to Download
+
+{% include/dataSubsets/_data_Imaging-Spec-Data-H5-2020.html %}
+
+*Remember* that the example dataset included here only has 1 out of every 4 bands
+included in a full NEON hyperspectral dataset (this makes it 1/4 the size!). When 
+we refer to bands in this tutorial, we will note both the band numbers for this 
+example dataset, and the band numbers for a standard NEON hyperspectral dataset 
+so that you can recreate these methods later with different inputs.
+
+***
+{% include/dataSubsets/_data_Imaging-Spec-Data-RGB-2020.html %}
+
+***
+{% include/_greyBox-wd-rscript.html %}
+
+### Recommended Skills
+
+We highly recommend you work through the 
+<a href="https://neonscience.org/hsi-hdf5-r" target="_blank"> *Introduction to Working with Hyperspectral Data in HDF5 Format in R* tutorial</a>
+before moving on to this tutorial.
+
```

Comment:
Can you add anything like, "or be comfortable with the A, B, and C concepts and performing X,Y, and Z skills."?  (ABC and XYZ should probably be connected to the learning objectives in the linked tutorial).  This allow for both beginning and more advanced users.

Reply:
Good idea - I have updated this information, and pointed the user to the intro to hyperspectral tutorial series as pre-requisite material if they don't already have the aforementioned skills/knowledge.